whois-api/src/main/java/net/ripe/db/whois/api/exceptions/UpgradeException.java

```diff
@@ -0,0 +1,12 @@
+package net.ripe.db.whois.api.exceptions;
+
+import javax.ws.rs.WebApplicationException;
+
+public class UpgradeException extends WebApplicationException {
+
```

Comment:
do you think UpgradeRequiredException is better name?

Reply:
I deleted the exception, we are returning directly the error